apps/extension/src/core/domains/mnemonics/types.ts

```diff
@@ -30,11 +30,21 @@ export declare type MnemonicDeleteRequest = {
   mnemonicId: MnemonicId
 }
 
+export type VerifierCertificateType = "new" | "talisman"
+
+export type RequestSetVerifierCertificateMnemonic = {
+  type: VerifierCertificateType
+  mnemonic?: string
+  mnemonicId?: string
+}
+
 export interface MnemonicMessages {
   // mnemonic message signatures
   "pri(mnemonic.unlock)": [MnemonicUnlockRequest, string]
   "pri(mnemonic.confirm)": [MnemonicConfirmRequest, boolean]
   "pri(mnemonic.address)": [RequestAddressFromMnemonic, string]
   "pri(mnemonic.rename)": [MnemonicRenameRequest, boolean]
   "pri(mnemonic.delete)": [MnemonicDeleteRequest, boolean]
+  "pri(mnemonic.validateMnemonic)": [string, boolean]
```

Comment:
might want to keep this one in accounts handler and rename it to validateSuri
goal is to validate if we can generate a valid address from a mnemonic + optional derivaiton path

i'm also adding a pri(accounts.validateDerivationPath) because of the new account screen where we don't always have a mnemonic